css/index.css

```diff
@@ -2,3 +2,248 @@
 This imports the `utility.css` file */
 @import url('./reset.css');
 @import url('./utility.css');
+@import url('https://fonts.googleapis.com/css2?family=Spartan:wght@700&display=swap');
+
+.theme1 {
+  --main-background-color: hsl(222deg 26% 31%);
+  --keypad-toggle-background-color: hsl(223deg 31% 20%);
+  --screen-background-color: hsl(224deg 36% 15%);
+  --key-backgound-color: hsl(225deg 21% 49%);
+  --key-shadow-color: hsl(224deg 28% 35%);
+  --primary-bg-color: hsl(30deg 25% 89%);
+  --key-shadow-keys: hsl(28deg 16% 65%);
+  --toggle-key-background-color: hsl(6deg 63% 50%);
+  --toggle-key-shadow-color: hsl(6deg 70% 34%);
+  --secondary-color: hsl(30deg 25% 89%);
+  --primary-color: hsl(0deg 0% 100%);
+  --tertiary-color: hsl(221deg 14% 31%);
+  --equals-color: hsl(0deg 0% 100%);
+  --dot-position: 8px;
+  --primary-text-color: hsl(224deg 51% 76%);
+  --key-hover: hsl(0deg 0% 100%);
+  --reset-dot-hover: hsl(6deg 93% 67%);
+}
+
+.theme2 {
+  --main-background-color: hsl(0deg 0% 90%);
+  --keypad-toggle-background-color: hsl(0deg 5% 81%);
+  --screen-background-color: hsl(0deg 0% 93%);
+  --key-backgound-color: hsl(185deg 42% 37%);
+  --key-shadow-color: hsl(185deg 58% 25%);
+  --primary-bg-color: hsl(45deg 7% 89%);
+  --key-shadow-keys: hsl(35deg 11% 61%);
+  --toggle-key-background-color: hsl(25deg 98% 40%);
+  --toggle-key-shadow-color: hsl(25deg 99% 27%);
+  --secondary-color: hsl(60deg 10% 19%);
+  --primary-color: hsl(0deg 0% 100%);
+  --tertiary-color: hsl(60deg 10% 19%);
+  --equals-color: hsl(0deg 0% 100%);
+  --dot-position: 25px;
+  --primary-text-color: hsl(186deg 41% 56%);
+  --key-hover: hsl(0deg 0% 100%);
+  --reset-dot-hover: hsl(25deg 100% 61%);
+}
+
+.theme3 {
+  --main-background-color: hsl(268deg 75% 9%);
+  --keypad-toggle-background-color: hsl(268deg 71% 12%);
+  --screen-background-color: hsl(268deg 71% 12%);
+  --key-backgound-color: hsl(281deg 89% 26%);
+  --key-shadow-color: hsl(285deg 91% 52%);
+  --primary-bg-color: hsl(273deg 47% 21%);
+  --key-shadow-keys: hsl(290deg 70% 36%);
+  --toggle-key-background-color: hsl(177deg 92% 70%);
+  --toggle-key-shadow-color: hsl(176deg 100% 44%);
+  --secondary-color: hsl(52deg 100% 62%);
+  --primary-color: hsl(0deg 0% 100%);
+  --tertiary-color: hsl(52deg 100% 62%);
+  --equals-color: hsl(198deg 20% 13%);
+  --dot-position: 45px;
+  --primary-text-color: hsl(280deg 57% 44%);
+  --key-hover: hsl(268deg 54% 44%);
+  --reset-dot-hover: hsl(177deg 100% 79%);
+}
+
+.input {
+  height: 30px;
+  width: 80px;
+  border-radius: 30px;
+  background-color: var(--bg-toggle-keypad);
+  position: relative;
```

Comment:
Can you please use the utility class for this

Reply:
I have done this